action.yml

```diff
@@ -75,8 +87,8 @@ runs:
     - uses: crazy-max/ghaction-import-gpg@v3
       if: ${{ inputs.gpg_private_key !== '' && inputs.gpg_private_key !== null }}
       with:
-        gpg-private-key: ${{ secrets.gpg_private_key }}
-        passphrase: ${{ secrets.gpg_passphrase }}
+        gpg-private-key: ${{ inputs.gpg_private_key }}
```

Comment:
These should be set as secrets

Reply:
@tundeaoni composite actions do not support `secrets`. 
https://stackoverflow.com/a/70111134/12472166

Reply:
@tundeaoni if we need to use secrets, we'll have to think of something else rather than composite actions. Moving forward and merging.